_posts/2021-06-28-Synchronizing-physical-clocks-among-different-UnetStack-based-modems.md

```diff
@@ -10,14 +10,14 @@ thumbnail: "assets/img/stdma/stdma-unet.jpg"
 tags: [howto, time synchronization, clock drift, accurate clocks, clock offset]
 ---
 
-All UnetStack enabled modems have a “real-time clock” to keep track of time using a precise quartz crystal. Based on the defined frequency of this quartz, software running on the modem can precisely monitor the internal physical time given by parameter `phy.time` in the modem. 
+All UnetStack enabled modems have a Real-time Clock (RTC) to keep track of modem's system time using a precise quartz crystal. A separate microsecond counter is provided by `phy.time` for `Physical` agent in UnetStack.
```

Comment:
This one still doesn't work for me. Why is important that RTC is precise or not?  It's `phy.time` that this article is about!

Reply:
Modified the introduction now to be more clear now.